Local docker environment setup (#30)
stefmolin/python-data-viz-workshop: README.md

    ![check env](./media/env_check.png)

> ### Docker installation

Reviewer: ```suggestion
### Docker Installation
```

> If you have Docker installed in your system (instructions [here](https://docs.docker.com/get-docker/)), you can set up the environment using `docker compose`. Run through the [local installation guide](#local-installation) (skip installing anaconda in step 0, as Docker takes care of installing this dependency), and instead of steps 4 and 5, do

Reviewer: ```suggestion
If you have Docker installed in your system (instructions [here](https://docs.docker.com/get-docker/)), you can set up the environment using `docker compose`. Run through the [local installation guide](#local-installation) (skip installing Anaconda in step 0, as Docker takes care of installing this dependency), and instead of steps 4 and 5, do
```

> If you have Docker installed in your system (instructions [here](https://docs.docker.com/get-docker/)), you can set up the environment using `docker compose`. Run through the [local installation guide](#local-installation) (skip installing anaconda in step 0, as Docker takes care of installing this dependency), and instead of steps 4 and 5, do

Reviewer: I would prefer to have a complete set of steps in each section and then around line 37 include a list of the options that you can click to get to each of these setup options. Otherwise you have to do a lot of jumping around between this section and the previous one.

4. Build the Docker image needed to run the Jupyter environment:

    ```shell
    docker compose build
    ```

> 5. Launch JupyterLab from within Docker:

Reviewer: Would it be more correct to say it this way?

```suggestion
5. Launch JupyterLab from within a Docker container:
```

    ```shell
    docker compose up
    ```

> You should be able to access the environment at [http://localhost:8888](http://localhost:8888). Follow through steps 6-7 normally.

Reviewer: I think you can just do this here:

```suggestion
You should be able to access the environment at <http://localhost:8888>. Follow through steps 6-7 normally.
```

> *Note: to terminate the Docker service once you're done, use `ctr+c` to terminate the command.*

Reviewer: ```suggestion
*Note: Once you're done, use `ctrl+c` to stop the Docker container.*
```
